detection-rules/callback_phishing_extensionless_rfc822_image.yml

```diff
@@ -0,0 +1,80 @@
+name: "Callback Phishing via extensionless rfc822 attachment "
+description: "This rule detects messages with an unknown file_type (extensionless) and a content_type of 'message/rfc822' that do not contain an email, but rather an image file with Callback Phishing indicators."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments,
+          .file_type == "unknown"
+          and .content_type == "message/rfc822"
+          and not any(file.explode(.),
```

Comment:
is this `not` supposed to be here?

Reply:
yep

Reply:
I can move it to the line below if that makes it more clear
doesn't change the functionality

Reply:
that should be better?